WordPress/WordPressUITests/Tests/MainNavigationTests.swift

```diff
@@ -17,26 +17,41 @@ class MainNavigationTests: XCTestCase {
         removeApp()
     }
 
-    func testTabBarNavigation() throws {
+    // We run into an issue where the People screen would crash short after loading.
+    // See https://github.com/wordpress-mobile/WordPress-iOS/issues/20112.
+    //
+    // It would be wise to add similar tests for each item in the menu (then remove this comment).
+    func testLoadsPeopleScreen() throws {
         XCTAssert(MySiteScreen.isLoaded(), "MySitesScreen screen isn't loaded.")
 
-        _ = try mySiteScreen
-            .tabBar.goToReaderScreen()
+        mySiteScreen.goToMenu()
+        mySiteScreen.goToPeople()
```

Comment:
And I think you're right in that the reason clicking on "People" item results in a loading loop is because of missing stubs:

![image](https://user-images.githubusercontent.com/17252150/218395388-5801fbeb-267b-488a-be2c-5810ff94bbf6.png)